roles/gui-tools/defaults/main.yml

```diff
@@ -94,3 +84,12 @@ i3_xresources_block: |
     gtk.font_name: Nerd Font Mono 10
     regolith.wallpaper.file: /opt/lalubuntu/screens/wallpaper.png
     regolith.wallpaper.options: stretched
+
+i3_touchpad_block: |
+    Section "InputClass"
+        Identifier "touchpad"
+        Driver "libinput"
+        MatchIsTouchpad "on"
+        Option "Tapping" "on"
```

Comment:
Let's only verify the intended line is present, otherwise we might break our users config... :)
So go with line present in file, before endSection, tapping ?